packages/sdk/src/types/internal/context.ts

```diff
@@ -513,7 +513,7 @@ export class InternalContext {
 		if (isNewUser) {
 			this.context.emitter.emit('user-joined', user);
 		} else {
-			this.context.emitter.emit('user-updated', user);
+			this.context.emitter.emit('user-update', user);

```

Comment:
>user-update [](start = 30, length = 11)

`user-updated` was correct. (This is different from the event you're renaming above).